library/core/src/slice/mod.rs

```diff
@@ -9,7 +9,7 @@
 use crate::cmp::Ordering::{self, Equal, Greater, Less};
```

Comment:
The changes to this file just get deleted in https://github.com/rust-lang/rust/pull/128254/commits/21ee15741d020022d3b6736b378240dd1a697743, they could probably be dropped from this PR so there is only one performance-relevant commit (unless https://github.com/rust-lang/rust/pull/128254 is expected to take longer).

Reply:
Done.